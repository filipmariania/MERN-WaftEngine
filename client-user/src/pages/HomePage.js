import React, { Component } from "react";
import { Link } from "react-router-dom";
import fb from "../img/fb.svg";
import tw from "../img/tw.svg";
import profilepic from "../img/profilepic.png";
import user1 from "../img/user1.png";
import user2 from "../img/user2.png";
import postImg1 from "../img/postImg1.png";
import postImg2 from "../img/postImg2.png";
import offer1 from "../img/offer1.jpg";
import offer2 from "../img/offer2.jpg";
import offer3 from "../img/offer3.jpg";

export default class HomePage extends Component {
  render() {
    return (
      <div className="col-md-9">
        <div className="row">
          <div className="col-md-8">
            <div className="post">
              <textarea placeholder="Hey, Thinking About Trips? Offer Now" />
              <button className="btn btn-light btn-sm mr-2">
                Start Date - End Date
              </button>
              <button className="btn btn-light btn-sm mr-2">From</button>
              <button className="btn btn-light btn-sm mr-2">To</button>
              <button className="btn btn-primary btn-sm float-right">
                Continue
              </button>
            </div>

            <div className="card mb-4">
              <div className="media p-2">
                <img alt="pics" className="maxWidth40" src={user1} />
                <div className="media-body pl-2">
                  <Link to="/" className="bold">
                    Hiking to Punhill
                  </Link>
                  <div className="text-sm">
                    <span>Est. Cost: USD 460</span> | <span>Days:5</span>
                  </div>
                </div>
              </div>
              <div className="card-img">
                <img alt="pics" className="responsive-img" src={postImg1} />
              </div>

              <div className="p-2">
                <div className="row">
                  <div className="col-md-6">
                    <span className="text-sm pr-2">Share</span>
                    <Link to="/" className="p-2">
                      <img alt="pics" src={fb} />
                    </Link>
                    <Link to="/" className="p-2">
                      <img alt="pics" src={tw} />
                    </Link>
                  </div>
                  <div className="col-md-6 text-right">
                    <Link to="/trip-detail" className="btn btn-link btn-sm">
                      View Trip Details
                    </Link>
                    <Link to="/guest-detail" className="btn btn-primary btn-sm">
                      Ask4Trip
                    </Link>
                  </div>
                </div>
              </div>
            </div>

            <div className="card mb-4">
              <div className="media p-2">
                <img alt="pics" className="maxWidth40" src={user2} />
                <div className="media-body pl-2">
                  <Link to="/" className="bold">
                    Jumla RaRa Trek
                  </Link>
                  <div className="text-sm">
                    <span>Est. Cost: USD 460</span> | <span>Days:5</span>
                  </div>
                </div>
              </div>
              <div className="card-img">
                <img alt="pics" className="responsive-img" src={postImg2} />
              </div>

              <div className="p-2">
                <div className="row">
                  <div className="col-md-6">
                    <span className="text-sm pr-2">Share</span>
                    <Link to="/" className="p-2">
                      <img alt="pics" src={fb} />
                    </Link>
                    <Link to="/" className="p-2">
                      <img alt="pics" src={tw} />
                    </Link>
                  </div>
                  <div className="col-md-6 text-right">
                    <Link to="/trip-detail" className="btn btn-link btn-sm">
                      View Trip Details
                    </Link>
                    <Link to="/guest-detail" className="btn btn-primary btn-sm">
                      Ask4Trip
                    </Link>
                  </div>
                </div>
              </div>
            </div>
          </div>

          <div className="col-md-4 pt-3">
            <h2>Latest Offers</h2>

            <div className="card-box">
              <div className="image-container">
                <img alt="pics" className="responsive-img" src={offer1} />
                <div className="overShadow">
                  <h4>
                    Everest Base Camp Trek via Gokyo Lakes and Cho La Pass
                  </h4>
                </div>
              </div>
              <div className="row">
                <div className="col-6">
                  <div className="text-sm pl-2">
                    Est. Cost
                    <br />
                    USD2990
                  </div>
                </div>
                <div className="text-sm col-6 text-right">
                  <div className="pr-2">
                    {" "}
                    Days
                    <br /> 18
                  </div>
                </div>
              </div>
            </div>

            <div className="card-box">
              <div className="image-container">
                <img alt="pics" className="responsive-img" src={offer2} />
                <div className="overShadow">
                  <h4>Mardi Trek</h4>
                </div>
              </div>
              <div className="row">
                <div className="col-6">
                  <div className="text-sm pl-2">
                    Est. Cost
                    <br />
                    USD2990
                  </div>
                </div>
                <div className="text-sm col-6 text-right">
                  <div className="pr-2">
                    {" "}
                    Days
                    <br /> 18
                  </div>
                </div>
              </div>
            </div>

            <div className="card-box">
              <div className="image-container">
                <img alt="pics" className="responsive-img" src={offer3} />
                <div className="overShadow">
                  <h4>Upper Mustang Trek</h4>
                </div>
              </div>
              <div className="row">
                <div className="col-6">
                  <div className="text-sm pl-2">
                    Est. Cost
                    <br />
                    USD2990
                  </div>
                </div>
                <div className="text-sm col-6 text-right">
                  <div className="pr-2">
                    {" "}
                    Days
                    <br /> 18
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    );
  }
}
