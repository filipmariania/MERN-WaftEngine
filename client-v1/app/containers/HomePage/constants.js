// export const LOAD_COMPANY_REQUEST = 'app/HomePage/LOAD_COMPANY_REQUEST';
// export const LOAD_COMPANY_SUCCESS = 'app/HomePage/LOAD_COMPANY_SUCCESS';
// export const LOAD_COMPANY_FAILURE = 'app/HomePage/LOAD_COMPANY_FAILURE';
