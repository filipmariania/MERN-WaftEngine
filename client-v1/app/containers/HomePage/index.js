import React, { Component } from 'react';
import { createStructuredSelector } from 'reselect';
import { compose } from 'redux';
import { connect } from 'react-redux';
import { withRouter } from 'react-router-dom';

import injectSaga from 'utils/injectSaga';
import injectReducer from 'utils/injectReducer';
import saga from './saga';
import reducer from './reducer';

/* eslint-disable react/prefer-stateless-function */

class HomePage extends Component {
  state = {};

  componentDidMount() {}

  handleChange = value => {
    this.setState({ value });
  };

  render() {
    const showModal = !!this.props.match.params.slug;

    return (
      <div>
        <React.Fragment>
          <div className="container">
            <h3>This is HomePage</h3>
          </div>
        </React.Fragment>
      </div>
    );
  }
}

const withReducer = injectReducer({ key: 'homePage', reducer });
const withSaga = injectSaga({ key: 'homePage', saga });

const mapStateToProps = createStructuredSelector({});

const mapDispatchToProps = dispatch => ({});

const withConnect = connect(
  mapStateToProps,
  mapDispatchToProps,
);
export default compose(
  withRouter,
  withReducer,
  withSaga,
  withConnect,
)(HomePage);
