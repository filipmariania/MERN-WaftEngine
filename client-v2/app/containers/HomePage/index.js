/*
 * HomePage
 *
 * This is the first thing users see of our App, at the '/' route
 *
 * NOTE: while this component should technically be a stateless functional
 * component (SFC), hot reloading does not currently support SFCs. If hot
 * reloading is not a necessity for you then you can refactor it and remove
 * the linting exception.
 */

import React from 'react';
import { FormattedMessage } from 'react-intl';
import messages from './messages';
import StaticContentDiv from '../../components/StaticContentDiv';
import SlickSlider from '../../components/SlickSlider';
import mainImage from './home.png';

/* eslint-disable react/prefer-stateless-function */
export default class HomePage extends React.PureComponent {
  render() {
    const { classes, category } = this.props;

    return (
      <React.Fragment>
        <img
          style={{ width: '100%', maxWidth: '1440px' }}
          src={mainImage}
          alt="template"
        />
        <SlickSlider slideKey="manoj-dai" />
      </React.Fragment>
    );
  }
}
