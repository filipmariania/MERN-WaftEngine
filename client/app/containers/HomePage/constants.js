export const LOAD_FOURORG_REQUEST = 'app/HomePage/SearchComponent/LOAD_FOURORG_REQUEST';
export const LOAD_FOURORG_SUCCESS = 'app/HomePage/SearchComponent/LOAD_FOURORG_SUCCESS';
export const LOAD_FOURORG_FAILURE = 'app/HomePage/SearchComponent/LOAD_FOURORG_FAILURE';

export const LOAD_ORG_REQUEST = 'app/HomePage/SearchComponent/LOAD_ORG_REQUEST';
export const LOAD_ORG_SUCCESS = 'app/HomePage/SearchComponent/LOAD_ORG_SUCCESS';
export const LOAD_ORG_FAILURE = 'app/HomePage/SearchComponent/LOAD_ORG_FAILURE';
