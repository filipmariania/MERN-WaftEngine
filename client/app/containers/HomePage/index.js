/*
 * HomePage
 *
 * This is the first thing users see of our App, at the '/' route
 *
 * NOTE: while this component should technically be a stateless functional
 * component (SFC), hot reloading does not currently support SFCs. If hot
 * reloading is not a necessity for you then you can refactor it and remove
 * the linting exception.
 */

import React from 'react';
import { FormattedMessage } from 'react-intl';
import messages from './messages';
import { Link } from 'react-router-dom';
import styled from 'styled-components';
import { Icon, Button, TextField, Paper } from '@material-ui/core';
import logo from 'assets/img/logo.png';
import background from 'assets/img/background.jpg';
import rest1 from 'assets/img/rest1.jpg';
import categoryIcon from 'assets/img/rest_thumb.png';
import AppBar from '@material-ui/core/AppBar';
import Tabs from '@material-ui/core/Tabs';
import Tab from '@material-ui/core/Tab';
import Typography from '@material-ui/core/Typography';
import { Search, CalendarToday, LocationOn, Timelapse, Call, Email, Wifi } from '@material-ui/icons';
import { withStyles } from '@material-ui/core/styles';
import aries from "assets/img/aries.png";


function TabContainer(props) {
    return (
        <Typography component="div" style={{ padding: 8 * 3 }}>
            {props.children}
        </Typography>
    );
}

const Masthead = styled.div`
  background: #333 url(${background}) no-repeat center center;
  height: 350px;
  padding-top: 20px;
  color: #fff;
  margin-bottom:50px;
`;


const CategoryItem = styled.div`
box-shadow: 0 5px 5px rgba(0, 0, 0, 0.2);
padding: 30px 50px;
margin-bottom:20px;
`;






/* eslint-disable react/prefer-stateless-function */
export default class HomePage extends React.Component {
    state = {
        value: 0,
    };

    handleChange = (event, value) => {
        this.setState({ value });
    };


    render() {
        const { value } = this.state;
        return (
            <section className="is-vertical">
                <header className="header header-main">
                    <div className="container">
                        <div className="logo">
                            <Link to="/">
                                <img src={logo} />
                            </Link>
                        </div>
                    </div>
                </header>

                <Masthead>
                    <div className="text-center">
                        <br />
                        <br />
                        <h2>Ask To Marina</h2>

                        <strong className="content-title">
                            BE A PART OF THE LOCAL SEARCH REVOLUTION
            </strong>
                        <br />
                        <div className="search-container">
                            <div className="row">
                                <TextField className="col-4"
                                    id="standard-select-category"
                                    select
                                    label="Select"
                                />

                                <TextField className="col-6" fullWidth label="Please Input" />
                                <Button size="small" className="col-2" variant="contained" color="primary">
                                    Search
                            </Button>
                            </div>
                            <div className="clearfix" />
                        </div>
                    </div>
                </Masthead>

                <div className="container">

                    <CategoryItem>
                        <h2>Categories</h2>
                        <div className="media-info">
                            <div className="img">
                                <img src={categoryIcon} /></div>
                            <div className="info">
                                <strong><a href="#">Restaurant</a> </strong>
                                <span className="designation">4606</span></div></div>

                    </CategoryItem>


                    <div>
                        <Paper>
                            <Tabs value={value} onChange={this.handleChange} indicatorColor="primary"
                                textColor="primary"
                                centered>
                                <Tab label="Recent Videos" />
                                <Tab label="About Us" />
                                <Tab label="Rashifal" />
                            </Tabs>
                        </Paper>
                        {value === 0 && <TabContainer><div className="text-center"><iframe width="560" height="315" src="https://www.youtube.com/embed/a_wuykzfFzE" frameborder="0" allowFullScreen /></div></TabContainer>}
                        {value === 1 && <TabContainer><div className="text-center"><div className="maxWidth500">A maggot wound is easily identifiable - A hole of any sort, a characteristic smell. You can actually see the maggots moving in the wound. But do not worry the treatment is simple. The vet cleans the area with antiseptic. They then applies Lorex­ene, or Maggocide, ointments specifically meant to kill maggots.</div></div> </TabContainer>}
                        {value === 2 && <TabContainer>
                            <div className="row">
                                <div className="col-4">
                                    <div className="card box-card is-always-shadow">
                                        <div className="card__header"><div className="clearfix">
                                            <span className="designation">मेष - चु, चे, चो, ला, लि, लु, ले, लो, अ (Aries)</span>
                                        </div></div>
                                        <div className="card__body">
                                            <div className="text item media-info">
                                                <div className="img"><img src={aries} height="60" width="60" />
                                                </div> <div className="info"><span className="designation">
                                                    Lorem ipsum dolor sit amet, consectetur
                                                                                                      adipisicing elit. Proin nibh augue
                                                  conseqaut nibbhi ellit ipsum consectetur.</span></div></div></div></div>
                                </div>
                            </div>
                        </TabContainer>}
                    </div>

                    <br />
                    <h2 className="text-center">Some of Organizations</h2>
                    <div className="card">
                        <div className="row">
                            <div className="col-3">
                                <img src={rest1} /></div>
                            <div className="col-9">
                                <br />
                                <h5>Thakali Restaurant</h5>
                                <CalendarToday />Opening Days:SUN, MON, TUE, WED, THU, FRI <br />
                                <LocationOn /> State 3 Bhaktapur Thimi,hanumante Bridge<br />
                                <Timelapse /> Opening Time:10AM-7PM<br />
                                <Call /> 977-9849242008<br />
                                <Email /> info@email.com<br />
                                <Wifi />http://website.com
          </div>
                        </div>
                    </div>

                </div>
                <div className="cta-block">
                    <div className="container">
                        <h2>Require something special?</h2>
                        <p>We might help you to build it better way!</p>
                        <Button variant="secondary" size="large">
                            <span>Send us some words</span></Button></div></div>






            </section>
        );
    }
}
