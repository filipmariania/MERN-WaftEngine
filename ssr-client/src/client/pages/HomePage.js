import React from 'react';
import { withRouter } from 'react-router-dom';

export class Home extends React.Component {
  // handleSearchSubmit = () => {
  //   this.props.history.push('/map');
  // };
  render() {
    return (
      <div className="container">
        <div className="row">
          <div className="col-md-3">
            <div className="media pt-4 mb-4">
              <img className="maxWidth40" src="/profilepic.png" alt="profile" />
              <div className="media-body pt-2 pl-2">Celina Jose</div>
            </div>

            <ul className="navList">
              <li>
                <a className="active" href="/">
                  Trip Feed
                </a>
              </li>
              <li>
                <a href="/">Upcoming Trips</a>
              </li>
              <li>
                <a href="/">Recent Trips</a>
              </li>
            </ul>
            <ul className="nav nav-horz">
              <li>About us </li>
              <li> | </li>
              <li>Terms of Use</li>
            </ul>
          </div>

          <div className="col-md-6">
            <div className="post">
              <textarea placeholder="Hey, Thinking About Trips? Offer Now" />
              <button className="btn btn-light btn-sm mr-2">Start Date - End Date</button>
              <button className="btn btn-light btn-sm mr-2">From</button>
              <button className="btn btn-light btn-sm mr-2">To</button>
              <button className="btn btn-primary btn-sm float-right">Continue</button>
            </div>

            <div className="card mb-4">
              <div className="media p-2">
                <img className="maxWidth40" src="user1.png" alt="user1" />
                <div className="media-body pl-2">
                  <a className="bold" href="/">
                    Hiking to Punhill
                  </a>
                  <div className="text-sm">
                    <span>Est. Cost: USD 460</span> | <span>Days:5</span>
                  </div>
                </div>
              </div>
              <div className="card-img">
                <img className="responsive-img" src="postImg1.png" alt="post1" />
              </div>

              <div className="p-2">
                <div className="row">
                  <div className="col-md-6">
                    <span className="text-sm pr-2">Share</span>
                    <a href="/" className="p-2">
                      <img src="/fb.svg" alt="fb" />
                    </a>
                    <a href="/" className="p-2">
                      <img src="/tw.svg" alt="tw" />
                    </a>
                  </div>
                  <div className="col-md-6 text-right">
                    <a className="btn btn-link btn-sm" href="/">
                      View Trip Details
                    </a>
                    <a href="/" className="btn btn-primary btn-sm">
                      Ask4Trip
                    </a>
                  </div>
                </div>
              </div>
            </div>

            <div className="card mb-4">
              <div className="media p-2">
                <img className="maxWidth40" src="/user2.png" alt="user2" />
                <div className="media-body pl-2">
                  <a className="bold" href="/">
                    Jumla RaRa Trek
                  </a>
                  <div className="text-sm">
                    <span>Est. Cost: USD 460</span> | <span>Days:5</span>
                  </div>
                </div>
              </div>
              <div className="card-img">
                <img className="responsive-img" src="/postImg2.png" alt="postImg2" />
              </div>

              <div className="p-2">
                <div className="row">
                  <div className="col-md-6">
                    <span className="text-sm pr-2">Share</span>
                    <a href="/" className="p-2">
                      <img src="/fb.svg" alt="fb" />
                    </a>
                    <a href="/" className="p-2">
                      <img src="/tw.svg" alt="tw" />
                    </a>
                  </div>
                  <div className="col-md-6 text-right">
                    <a className="btn btn-link btn-sm" href="/">
                      View Trip Details
                    </a>
                    <a href="/" className="btn btn-primary btn-sm">
                      Ask4Trip
                    </a>
                  </div>
                </div>
              </div>
            </div>
          </div>

          <div className="col-md-3 pt-3">
            <h2>Latest Offers</h2>

            <div className="card-box">
              <div className="image-container">
                <img className="responsive-img" src="/offer1.jpg" alt="offer1" />
                <div className="overShadow">
                  <h4>Everest Base Camp Trek via Gokyo Lakes and Cho La Pass</h4>
                </div>
              </div>
              <div className="row">
                <div className="col-6">
                  <div className="text-sm pl-2">
                    Est. Cost
                    <br />
                    USD2990
                  </div>
                </div>
                <div className="text-sm col-6 text-right">
                  <div className="pr-2">
                    {' '}
                    Days
                    <br /> 18
                  </div>
                </div>
              </div>
            </div>

            <div className="card-box">
              <div className="image-container">
                <img className="responsive-img" src="/offer2.jpg" alt="offer2" />
                <div className="overShadow">
                  <h4>Mardi Trek</h4>
                </div>
              </div>
              <div className="row">
                <div className="col-6">
                  <div className="text-sm pl-2">
                    Est. Cost
                    <br />
                    USD2990
                  </div>
                </div>
                <div className="text-sm col-6 text-right">
                  <div className="pr-2">
                    {' '}
                    Days
                    <br /> 18
                  </div>
                </div>
              </div>
            </div>

            <div className="card-box">
              <div className="image-container">
                <img className="responsive-img" src="/offer3.jpg" alt="offer3" />
                <div className="overShadow">
                  <h4>Upper Mustang Trek</h4>
                </div>
              </div>
              <div className="row">
                <div className="col-6">
                  <div className="text-sm pl-2">
                    Est. Cost
                    <br />
                    USD2990
                  </div>
                </div>
                <div className="text-sm col-6 text-right">
                  <div className="pr-2">
                    {' '}
                    Days
                    <br /> 18
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    );
  }
}

export default {
  component: withRouter(Home),
};
