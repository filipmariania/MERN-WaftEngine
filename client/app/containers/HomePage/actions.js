import * as types from './constants';

export const loadFourorgRequest = payload => ({
  type: types.LOAD_FOURORG_REQUEST,
  payload,
});
export const loadFourorgSuccess = payload => ({
  type: types.LOAD_FOURORG_SUCCESS,
  payload,
});
export const loadFourorgFailure = payload => ({
  type: types.LOAD_FOURORG_FAILURE,
  payload,
});

export const loadOrgRequest = payload => ({
  type: types.LOAD_ORG_REQUEST,
  payload,
});
export const loadOrgSuccess = payload => ({
  type: types.LOAD_ORG_SUCCESS,
  payload,
});
export const loadOrgFailure = payload => ({
  type: types.LOAD_ORG_FAILURE,
  payload,
});
