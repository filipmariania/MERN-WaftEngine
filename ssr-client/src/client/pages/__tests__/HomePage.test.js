import React from 'react';
import { create } from 'react-test-renderer';
import { Home } from '../HomePage';

test('snapshot', () => {
  const c = create(<Home />);
  expect(c.toJSON()).toMatchSnapshot();
});
