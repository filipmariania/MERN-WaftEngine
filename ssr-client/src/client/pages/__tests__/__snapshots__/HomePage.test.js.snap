// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<div
  className="center-align"
  style={
    Object {
      "marginTop": "200px",
    }
  }
>
  <h3>
    Welcome
  </h3>
  <form
    onSubmit={[Function]}
  >
    <div
      className="row"
    >
      <div
        className="input-field col s3"
      >
        <label
          htmlFor="latitudeSearch"
        >
          Latitude
          <input
            id="latitudeSearch"
            name="latitude"
            onChange={[Function]}
            placeholder="latitude"
            value=""
          />
          <span
            className="helper-text"
          />
        </label>
      </div>
      <div
        className="input-field col s3"
      >
        <label
          htmlFor="longitudeSearch"
        >
          Longitude
          <input
            id="longitudeSearch"
            name="longitude"
            onChange={[Function]}
            placeholder="longitude"
            value=""
          />
          <span
            className="helper-text"
          />
        </label>
      </div>
      <div
        className="input-field col s3"
      >
        <label
          htmlFor="distanceSearch"
        >
          Distance in Meter
          <input
            id="distanceSearch"
            name="distance"
            onChange={[Function]}
            placeholder="distance"
            value=""
          />
          <span
            className="helper-text"
          />
        </label>
      </div>
      <div
        className="col s3"
      >
        <label
          htmlFor="searchBtn"
        >
           
          <button
            className="btn"
            id="searchBtn"
          >
            Submit
          </button>
        </label>
      </div>
    </div>
  </form>
</div>
`;
